City-of-Helsinki/kukkuu-admin
# Chore | Use GitHub actions instead of Travis

## Description

Travis has a new pricing model and the free plan does not have enough
tokens to support the City of Helsinki's needs throughout a month.

This PR adds configurations that run linting, build and tests on PRs and when new commits are pushed to either develop or master. It also configures browser tests to be rain daily against the staging server.

## Context

Without this change we do not have a working CI.

## Testing instructions

You can check that the workflows pass here: https://github.com/City-of-Helsinki/kukkuu-admin/actions

I forced the browser test workflow to run once by adding a `on.pull_request` setting that I since removed. Because it has the `workflow_dispatch` setting, it should be runnable manually once these changes are merged into develop.
